pom.xml

```diff
@@ -668,6 +669,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
+                    <release>${jdk-release-version}</release>
                     <source>${jdk-version}</source>
                     <target>${jdk-version}</target>
```

Comment:
source and target can be removed when switching to jdk11. Currently this would cause compilation to fail on jdk8